c2_utils/src/mfx_va_allocator.cpp

```diff
@@ -595,43 +553,6 @@ mfxStatus MfxVaFrameAllocator::CreateSurfaceFromGralloc(const IMfxGrallocModule:
     attribs[1].flags = VA_SURFACE_ATTRIB_SETTABLE;
     attribs[1].value.type = VAGenericValueTypePointer;
     attribs[1].value.value.p = (void *)&surfExtBuf;
-#else
-    VADRMPRIMESurfaceDescriptor desc;
-    MFX_ZERO_MEMORY(desc);
-
-    desc.fourcc = va_fourcc;
-    desc.width = width;
-    desc.height = height;
-    desc.num_objects = 1;
-    desc.objects[0].fd = info.prime;
-    desc.objects[0].size = decode_target ? info.pitches[0] * ((height + 31) & ~31) * 1.5 : info.pitches[0] * ((height + 15) & ~15) * 1.5;
-    if (HAL_PIXEL_FORMAT_NV12_Y_TILED_INTEL == info.format || HAL_PIXEL_FORMAT_P010_INTEL == info.format)
```

Comment:
Why do we have to remove this part?

Thanks,
Tianmi

Reply:
Got it wrong, my local branch is messed up, will fix it. sorry...